mysql-test/suite/galera/t/galera_sst_xtrabackup-v2_super_read_only.test

```diff
@@ -1,26 +1,97 @@
 #
-# This test checks the pxc-encrypt-cluster-traffic option (auto SSL config).
-# Initial SST happens via xtrabackup, so there is not much to do in the body of the test
+# This test checks that the cluster starts up (without error)
+# if super-read-only=ON.  This will check server startup
+# as well as the SST process.
 #
 
 --source include/big_test.inc
 --source include/galera_cluster.inc
 
+--let $wait_condition = SELECT VARIABLE_VALUE = 2 FROM performance_schema.global_status WHERE VARIABLE_NAME = 'wsrep_cluster_size';
+--source include/wait_condition.inc
+
 SELECT 1;
 
+# --------------------------------
+# Test 1: Test that the cluster starts up
+# --------------------------------
+
+--connection node_2
+--echo [connection node_2]
+
+--let $NODE2_CONFIG_FILE = $MYSQLTEST_VARDIR/tmp/galera_sst_xtrabackup-v2_super_read_only.test.cnf
+--copy_file $MYSQLTEST_VARDIR/my.cnf $NODE2_CONFIG_FILE
+--append_file $NODE2_CONFIG_FILE
+	[mysqld]
+	super_read_only=ON
+EOF
+
+# --------------------------------
+# Test 1a: Restart node2 (no sst)
+# --------------------------------
+
+# restart the server with super-read-only=ON (no SST)
+--echo "# restarting node2 (no SST)"
+--let $restart_hide_args=1
+--let $restart_parameters = "restart: --defaults-file=$NODE2_CONFIG_FILE"
+--source include/restart_mysqld.inc
+--source include/wait_until_connected_again.inc
```

Comment:
Wouldn't just this be enough? Is it really required to create a temporary config file?
```
+ --echo "# restarting node2 (no SST)"
+ --let $restart_parameters = "restart: --super-read-only=ON"
+ --source include/restart_mysqld.inc
```

Reply:
see the comment below (it matters more for the SST process).